edx_event_bus_kafka/internal/consumer.py

```diff
@@ -303,25 +308,18 @@ def _consume_indefinitely(self):
         finally:
             self.consumer.close()
 
-    def consume_indefinitely(self, offset_timestamp=None):
+    def consume_indefinitely(self):
         """
         Consume events from a topic in an infinite loop.
-
-        Arguments:
-            offset_timestamp (datetime): Optional and deprecated; if supplied, calls
-                ``reset_offsets_and_sleep_indefinitely`` instead. Relying code should
-                switch to calling that method directly.
         """
-        # TODO: Once this deprecated argument can be removed, just
-        # remove this delegation method entirely and rename
-        # `_consume_indefinitely` to no longer have the `_` prefix.
-        if offset_timestamp is None:
```

Comment:
[thinking out loud] I had been hoping that we could fully remove the `offset_timestamp` from the consumer management command, as we'd been moving towards with this deprecation. (Resetting offsets is a very different operation from consuming, so we'd been hoping to separate them further.) I think we'll eventually want to have a separate management command that's just used for replay, but that that's out of scope here. We can always make a separate breaking change later in combination with a deeper refactor.